LDAR_Sim/src/initialization/sites.py

```diff
@@ -39,46 +65,51 @@ def generate_sites(program, in_dir):
         [type]: [description]
     """
     # Read in the sites as a list of dictionaries
-    with open(in_dir / program['infrastructure_file']) as f:
-        sites = [{k: v for k, v in row.items()}
-                 for row in csv.DictReader(f, skipinitialspace=True)]
+    sites_in = pd.read_csv(in_dir / program['infrastructure_file'], index_col='facility_ID')
+    # Add facility ID back into object
+    sites_in['facility_ID'] = sites_in.index
+    sites = sites_in.to_dict('index')
+
```

Comment:
Adding the index column back into the pandas dataframe feels weird  - but I'm assuming this is necessary to get this to work properly.

Reply:
Yeah, That was the most voted Stack Exchange way of keeping the facility_ID as the index and a column. The other option is setting the index as the facility_ID column after the fact. Lets keep that in the back of the mind though I aint a Pandas expert